experimental/python/databricks/bundles/catalog/_models/volume_type.py

```diff
@@ -0,0 +1,14 @@
+from enum import Enum
```

Comment:
[separate PR] It would be nice have generated comment there and in other generated files.

Reply:
👍 makes sense